src/test/enginesynctest.cpp

```diff
@@ -2146,20 +2146,125 @@ TEST_F(EngineSyncTest, UserTweakPreservedInSeek) {
 
     // Now, the locations are much more different - but the beat distance appears
     // to be the same because beat distance CO hides the user offset.
-    EXPECT_NEAR(0.2269025,
+    EXPECT_NEAR(0.1990531,
```

Comment:
I am not 100% sure that the change in these values is right, but I did change the seek phase code so it's plausible that this is right

Reply:
At least it is suspicious. It would be great if we can calc here the expected result. 
For now it is fine for me to test the case manual and it if it is still OK let go for it. 
I think this was the way we have found the constant originally.

Reply:
This was may be wrong initially, In case I seek a track with a user offset in 2.3 branch, the offset is ignored during the seek and than dragged back into phase. This is fixed here. Thank you.